include/openenclave/bits/types.h

```diff
@@ -224,4 +224,17 @@ typedef struct _oe_asymmetric_key_params
     size_t user_data_size;
 } oe_asymmetric_key_params_t;
 
+/**
+ * This struct defines a datetime up to 1 second precision.
+ */
+typedef struct _oe_datetime
+{
+    uint32_t year;    /* format: 1970, 2018, 2020 */
```

Comment:
>uint32_t [](start = 4, length = 8)

Consider uint16_t for all.

Reply:
Since this is already published, please ignore the comment.

---
In reply to: [344977176](https://github.com/openenclave/openenclave/pull/2298#discussion_r344977176) [](ancestors = 344977176)